code/__DEFINES/cooldowns.dm

```diff
@@ -41,6 +41,9 @@
 //car cooldowns
 #define COOLDOWN_CAR_HONK "car_honk"
 
+//wheelchair bell cooldown
+#define COOLDOWN_BELL "bell_cooldown"
```

Comment:
I don't think this needs a cooldown key at all and you should just use a standard cooldown variable on the bell itself.